Modules/System/BLOB Storage/src/TempBlobListImpl.Codeunit.al

```diff
@@ -8,9 +8,9 @@ codeunit 4109 "Temp Blob List Impl."
     Access = Internal;
 
     var
-    #pragma warning disable AA0073
+#pragma warning disable AA0073
```

Comment:
Formatting is weird here

Reply:
That's AL Language extension autoformatting. And as far as I can see, the pragma instruction is always aligned on the left without indentation.

Reply:
That formatting was reported on the AL Language Repository, but it wasn't solved yet.
See Issue https://github.com/microsoft/AL/issues/6530